### PR TITLE
modified entities.md

### PR DESCRIPTION
**Pull request by anandhi-iyappan**
- Modified entities.md to the new model changes
- Deleted concepts.md
- Deleted modules.md
- Modified standards.md

### CODE REVIEW

File: user_docs/metadata/overview.md [hunk 1]
```diff
@@ -13,9 +13,8 @@
...
...ema broadly differentiates the research and administrative aspects of the metadata model - the research metadata aims at maximising the reusability and FAIRness of the data while the administrative metadata The administrative metadata focusses on managing the resources such as creation or acquisition of the data, rights management, and disposition. The research metadata classes include Individual, Biospcimen/Sample, Experiment, Experiment methods, Analysis and Analysis methods. The administrative metadata captures Dataset, Data Access Committee (DAC) and Data Access Policy (DAP), Study/Publication.  Furthermore the model also differentiates between three file types, which are different classes controlled by ranges of file types - namely (i) research data files is which defined as the digital entity resulting from the measurement or sequencing of a sample such as FASTQ, RAW, IDAT (ii) processed data files defined as files that result from an analysis, alignment or computation processing step of a research data file such as BAM, VCF and (iii) supporting files are auxiliary files that provide further structured information about an individual, experiment or analysis such as JSON, TXT.
```
Comment (mauerk) [inline user_docs/metadata/overview.md:18]:
```suggestion
Classes in the schema can be grouped into **Research Metadata** and **Administrative Metadata** based on the information they capture. The **Research Metadata** aims at maximising the reusability and FAIRness of the data, while the **Administrative Metadata** focuses on managing the resources, such as creation or acquisition of the data, rights management, and disposition. The Research Metadata classes include *Individual*, *Biospcimen/Sample*, *Experiment*, *Experiment Method*, *Analysis* and *Analysis Method*. The Administrative Metadata captures *Dataset*, *Data Access Policy*, *Data Access Committee*, *Publication*, and *Study*.  

The model also differentiates between three file types:
- **Research Data File**: A file which results from the omics experiment, such as sequencing of a sample.
- **Process Data File**: A file that is generated as output from an analysis performed on a *Research Data File*, such as alignment or processing.
- **Supporting File**: A file that provides further information about an *Individual*, *Experiment Method* or *Analysis Method*. These could be unstructured protocols or structured information, such as Phenopackets or BioCompute Objects.
```

File: user_docs/metadata/standards.md [hunk 2]
```diff
@@ -12,54 +12,23 @@ Thousands of standards, ontologies and vocabularies have been developed for a va
...
-
-### **Minimum Information about a Proteomic Experiment (MIAPE)**
-The [MIAPE](https://www.psidev.info/miape) guidelines were founded by the [Human Proteome Organization's Proteomics Standards Initiative](https://www.hupo.org/Proteomics-Standards-Initiative-(PSI)) to develop guidelines for reporting the use of techniques such as gel electrophoresis and mass spectrometry.  One of the main objectives of the MIAPE is to increase the value derived by the scientific community from ongoing experimentation in proteomics. This is established through community processes that support sharing, dissemination and reanalysis of datasets, and that assist in establishing and promoting best practice in specific technical areas. The most important MI standards for genomics are listed below.
-
-### **Minimum Information about a high-throughput Nucleotide Sequencing Experiment (MINSEQE)**
```
Comment (mauerk) [inline user_docs/metadata/standards.md:40]: Can we restore this?

**Review by mauerk — CHANGES_REQUESTED** [review]
- please make sure that entity names are always in upper case and in italics ("*Experiment Method*" instead of "Experiment methods")
- the protocol entities are in singular ("Experiment Method" instead of "Experiment methods")
- abbreviations such as "ID" or "EGA" need to be upper case
- use double line breaks to generate new paragraphs to make the content more easily readbale, especially towards the end of the entities.md file
- deleted files also need to be deleted in the mkdocs.yaml page tree so they do not appear as 404 error in the generated page (run `mkdocs serve` in VS Code to view the page in your browser)
- the data dictionary needs to be updated

File: user_docs/metadata/standards.md (hunk 2 above)
Comment (sbilge) [reply user_docs/metadata/standards.md:40]: MINSEQE restored

File: user_docs/metadata/standards.md [hunk 3]
```diff
@@ -75,14 +44,8 @@ The [Mondo Disease Ontology (Mondo)](https://obofoundry.org/ontology/mondo) prov
...
-
 ### **Systematic Nomenclature of Medicine Clinical Terms**
```
[the remark this answers is not in the document]
Comment (sbilge) [reply user_docs/metadata/standards.md:47]: Systematic Nomenclature of Medicine Clinical Terms is removed

File: user_docs/metadata/entities.md [hunk 4]
```diff
@@ -4,101 +4,152 @@ This section provides an overview on what metadata elements are captured with th
...
 
-## **Sample**
+The Individual entity is linked to the *Sample* entity.
 
-GHGAs *Sample* metadata can be separated into three distinct entities: *Sample*, *Biospecimen* and *Condition*. Both the *Sample* and *Biospecimen* entities provide the data submitter with options to deposit metadata that allows for deeper insight into the characteristics of samples and biospecimen. The *Condition* allows to further define the state of the samples and to group samples within a study accordingly. The following paragraph gives a definition of what a sample, biospecimen or condition is in the context of GHGAs metadata schema.
+### **Individual Supporting File**
 
-A *Sample* is defined as a limited quantity of something to be used for testing, analysis, inspection, investigation, demonstration, or trial use. A sample is prepared from a biospecimen (isolate or tissue).
+If the data submitters have additional information to provide about the individual, then they can make use of the Individual supporting file class which requires the following information -  file format, file name, dataset associated with this file as well as if the file is included as part of the submission.
```
Comment (mauerk) [inline user_docs/metadata/entities.md:23]:
```suggestion
If the data submitters have additional information to provide about the *Individual*, then they can make use of the *Individual Supporting File* class which requires the following information -  file format, file name, dataset associated with this file as well as if the file is included as part of the submission. 
```

File: user_docs/metadata/entities.md [hunk 5]
```diff
@@ -4,101 +4,152 @@ This section provides an overview on what metadata elements are captured with th
...
+
+The *Sample* entity requires data submitters to provide the name, description, whether the sample is case or control, as well as the alias to the *Individual* from which the *Sample* was derived. In addition to the mandatory information, the data submitter is recommended to provide information about the storage, whether the sample can be classified as  diseased or healthy, and biological replicate information. Other optional fields include the type of the sample and an EGA accession ID. 
+
+The *Biospecimen* entity requires data submitters to provide information about the age of the individual at the time of sampling and the biospecimen tissue. The recommended properties include a name for the biospecimen, its type, the vital status of the individual at the time of sampling, and the biospecimen isolation and storage.
+
+### **Experiment**
+
+An *Experiment* is an investigation that consists of a coordinated set of actions and observations designed to generate data with the goal of verifying, falsifying, or establishing the validity of a hypothesis. The *Experiment* enttity describes the experimental setup or a protocol used for performing an omics experiment.
```
Comment (mauerk) [inline user_docs/metadata/entities.md:41]:
```suggestion
An *Experiment* is an investigation that consists of a coordinated set of actions and observations designed to generate data with the goal of verifying, falsifying, or establishing the validity of a hypothesis. The *Experiment* entity describes the experimental setup or a protocol used for performing an omics experiment. 
```

File: user_docs/metadata/entities.md [hunk 6]
```diff
@@ -4,101 +4,152 @@ This section provides an overview on what metadata elements are captured with th
...
+
+The *Biospecimen* entity requires data submitters to provide information about the age of the individual at the time of sampling and the biospecimen tissue. The recommended properties include a name for the biospecimen, its type, the vital status of the individual at the time of sampling, and the biospecimen isolation and storage.
+
+### **Experiment**
+
+An *Experiment* is an investigation that consists of a coordinated set of actions and observations designed to generate data with the goal of verifying, falsifying, or establishing the validity of a hypothesis. The *Experiment* enttity describes the experimental setup or a protocol used for performing an omics experiment. 
+
+The *Experiment* entity is linked to the *Sample* which implies one or more samples associated with this Experiment and the *Experimental Method* which implies one or more Experimental methods associated with this Experiment. The outcome of an *Experiment* results in *Research Data File*.
```
Comment (mauerk) [inline user_docs/metadata/entities.md:43]:
```suggestion
The *Experiment* entity is linked to the *Sample* which implies one or more samples associated with this Experiment and the *Experiment Method* which implies one or more experiment methods associated with this Experiment. The outcome of an *Experiment* results in *Research Data File*.
```

File: user_docs/metadata/entities.md [hunk 7]
```diff
@@ -4,101 +4,152 @@ This section provides an overview on what metadata elements are captured with th
...
 
-The *Sequencing Experiment* entity is used to group *Sequencing Processes*, *Library Preparation Protocols* and *Sequencing Protocols* within one *Sequencing Experiment*. Via the *Sequencing Process* and *Condition* entities, it is then linked to *Study*, *Sample* and *File*.
+#### **Analysis metadata properties**
 
-The *Sequencing Process* captures the technical parameters that were used to generate output from a *Sample* during a *Sequencing Experiment*.
+The data submitter is required to provide information about the *Analysis* title, the *Analysis Method* that is associated with this *Analysis* and the *Research Data Files* that this *Analysis* used as input to generate *Process Data Files*.
 
-The *Sequencing Protocol* entity gives a variety of properties that a data submitter can submit. Mandatory properties are the sequencing alias, the description of the sequencing protocol and the provision of the instrument model with which the sequencing was done. Optional properties include the offset, read and size for cell and umi barcodes, flow cell id and type, read length, target coverage, and the type of the used sequencing protocol.
+### **Analysis Methods**
```
Comment (mauerk) [inline user_docs/metadata/entities.md:79]:
```suggestion
### **Analysis Method**
```

File: user_docs/metadata/entities.md [hunk 8]
```diff
@@ -4,101 +4,152 @@ This section provides an overview on what metadata elements are captured with th
...
+#### **Analysis Methods metadata properties**
 
-## **File**
+For this submission, the data submitter is required to provide information about the *Analysis Method* name, a description, type, the workflow name, workflow repository and workflow DOI. Furthermore, it is also recommended to provide information about the workflow version, workflow tasks, parameters and software versions.
 
-At the core of GHGA is the deposition of raw files that have been generated while carrying out an experiment. These files also have to be annotated with metadata, in order to give data requesters more information on what files have been deposited at GHGA by the data submitter. Therefore this metadata will also be used by the user interface of GHGAs data portal to provide not only information on how many files are contained within a dataset, but also information on file size, file formats, checksums and file types.
+### **Analysis Method Supporting File**
 
-The files deposited at GHGA, and their metadata, will always link to an experiment entity.
+For any additional information pertaining to the *Analysis Methods* entity, the data submitter can use the *Analysis Method Supporting File*.
```
Comment (mauerk) [inline user_docs/metadata/entities.md:89]:
```suggestion
For any additional information pertaining to the *Analysis Method* entity, the data submitter can use the *Analysis Method Supporting File*.
```

**Review by mauerk — CHANGES_REQUESTED** [review]

**Review by mauerk — APPROVED** [review]

**Pull request merged**